src/types.d.ts

```diff
@@ -0,0 +1,11 @@
+export interface DropcssOptions {
+    html: string;
+    css: string;
+
+    shouldDrop?: (selector: string) => boolean;
+    didRetain?: (selector: string) => void;
+
+    keepText?: boolean;
```

Comment:
heh, i dont think this is actually used. i may have had plans to implement it at some point, but nothing is there currently :thinking: 

https://github.com/leeoniya/dropcss/blob/cf9d1264fa048b908baedec7be607538c87a3fea/src/html.js#L179

Reply:
Yep! You're right